faker/providers/misc/__init__.py

```diff
@@ -70,6 +71,13 @@ def language_code(cls):
         return cls.random_element(cls.language_codes)
 
     @classmethod
+    def uuid(cls):
```

Comment:
uuid3 and uuid5 both require some argument, and uuid1 is very insecure, so
I wouldn't bother. But it's really up to you and what you think it's useful
:)

On Tue, Jun 2, 2015 at 9:50 AM, Jared Culp <notifications@github.com> wrote:

> In faker/providers/misc/__init__.py
> <https://github.com/joke2k/faker/pull/252#discussion_r31530734>:
>
> > @@ -70,6 +71,13 @@ def language_code(cls):
> >          return cls.random_element(cls.language_codes)
> >
> >      @classmethod
> > +    def uuid(cls):
>
> Sure! Should I add methods for the other uuid types?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joke2k/faker/pull/252/files#r31530734>.
>